MetaMask/metamask-extension
# Part of 17670: Replace Typography with Text confirm-approve-content.component.js and home.component.js

## Explanation
Replace Typography with Text `confirm-approve-content.component.js` and `home.component.js`

Part of: #17670
## Screenshots/Screencaps
### Before
<img width="1220" alt="Before approve-content" src="https://user-images.githubusercontent.com/53999998/223711902-a5d61f72-739b-4810-af80-7400530d2657.png">

### After
<img width="1224" alt="After approve-content" src="https://user-images.githubusercontent.com/53999998/223711923-b8bc50ec-9f04-4f37-a927-d1491a874488.png">

## Manual Testing Steps

- Go to the latest build of storybook on this PR
- Follow path `Pages/ConfirmApprove/ConfirmApproveContent` in the side panel
- Check `ConfirmApproveContent` story against develop storybook

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
